### ISSUE TITLE
SvgQRCode sets size too large if quiet zone is omitted when using pixelsPerModule

### ISSUE THREAD

**Issue by FStolte** [codebude/QRCoder#285]
## Type of issue
Bug

## Expected Behavior
When using the SvgQRCode renderer to create a QR-Code with x pixels per module and no quiet zones (i.e. `SvgQRCode.GetGraphic(5, "#000000", "#FFFFFF", false)`), the size of modules should be as specified.

## Current Behavior
The size of the whole QR-Code is the same as it would be with quiet zones, and therefore the modules are slightly too big.

## Possible Solution
In the `GetGraphics()` overloads that take a `pixelsPerModule` parameter, account for the missing quiet zone when creating the `Size`, e.g.
```
public string GetGraphic(int pixelsPerModule)
{
   var offset = drawQuietZones ? 0 : 4;
   var size = this.QrCodeData.ModuleMatrix.Count - (offset * 2)`
    var viewBox = new Size(size, size);
    return this.GetGraphic(viewBox, Color.Black, Color.White);
}
```

## Your Environment

* Version used:  1.4.1 NuGet package
* Environment: .NET 5

**Comment by codebude** [comment]
Hi @FStolte ,

thanks for your valuable input. Just one question: Shouldn't be the overloads to be changed that ones, which have the "pixelsPerModule" parameter **and** the "drawQuietZones" parameter? I'm asking because the overload you've shown as "possible solution" is the overload without the "drawQuietZones" parameter. From my understanding this overload draws the quietZones anyway, so that especially this overload isn't affected from the bug you're describing. Or do I oversee something?

Long story short: From my understanding the following overloads should be changed, correct?

- https://github.com/codebude/QRCoder/blob/master/QRCoder/SvgQRCode.cs#L24
- https://github.com/codebude/QRCoder/blob/master/QRCoder/SvgQRCode.cs#L30

**Comment by FStolte** [comment]
You’re right, it’s only the overloads that take both pixelsPerModule and drawQuietZones.

I was trying to give a minimal example and I seem to have simplified it a bit too much 😅

**Comment by codebude** [comment]
Hi @FStolte ,

bug was fixed with this commit: https://github.com/codebude/QRCoder/commit/0a3c4d346cccd65fa8f6fd48b5bfade2c015fe4e
Feel free to re-open the ticket, if this solution doesn't fit your needs. Otherwise the changes will appear with the next version/release on Nuget.
